modules/openapi-generator/src/main/resources/Java/libraries/feign/pojo.mustache

```diff
@@ -76,6 +76,7 @@ public class {{classname}} {{#parent}}extends {{{.}}} {{/parent}}{{#vendorExtens
   private {{{datatypeWithEnum}}} {{name}}{{#defaultValue}} = {{{.}}}{{/defaultValue}};
   {{/isContainer}}
   {{^isContainer}}
+  {{>nullable_var_annotations}}
```

Comment:
Hi @nvervelle, thanks for the PR!

For `feign`, the annotations on the field are currently only generated if the field is not a container. Moving the `{{>nullable_var_annotations}}` to line 75 should fix that. Other than that the PR looks good to me.

Reply:
Hi @martin-mfg !

Thanks for the comment. I modified the template for feign and updated the PR accordingly (I edited each commit so that I still have one commit for the manual changes, and one commit for the changes made by the script)

Reply:
I can't merge PRs. We'll have to wait for `@wing328` to merge this.

Thanks for the heads-up in case further input from you is needed. But hopefully the PR can simply be merged without further changes.

Reply:
^ @wing328 the mention didn't work. @nvervelle Unfortunately this PR needs a rebase now, sorry :(

Reply:
@martin-mfg , @joscha , @wing328 : I've rebased the PR.